client/landing/stepper/declarative-flow/copy-site.tsx

```diff
@@ -17,6 +17,7 @@ import {
 	AssertConditionState,
 	Flow,
 	ProvidedDependencies,
+	StepperStep,
```

Comment:
Nit: this should be a type import. Some of the others above also look like they should be type imports.
```suggestion
	type StepperStep,
```